server/status/recorder.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 const (
-	storeTimeSeriesNameFmt = "cr.store.%d.%s"
+	storeTimeSeriesNameFmt = "cr.store.%s.%d"
```

Comment:
Why don't you actually document what the values are in a comment?

Also, are you sure you don't want to encode that number with an ordered encoding?